lib/main.dart

```diff
@@ -1,9 +1,29 @@
+import 'dart:convert';
+
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:movie/flavors/env_enum.dart';
 import 'package:presentation/app/my_app.dart';
+import 'flavors/config_data.dart';
+import 'flavors/config_presentation.dart';
 import 'di/app_injector.dart';
 
-void main() {
+void mainCommon({Environment env = Environment.production}) async {
   WidgetsFlutterBinding.ensureInitialized();
-  initAppInjector();
-  runApp(const MyApp());
-}
\ No newline at end of file
+  final currentEnv = env == Environment.production ? 'prod' : 'sandbox';
```

Comment:
use `describeEnum` method

Reply:
or maybe  ? Environment.production.name
      : Environment.sandbox.name;

Reply:
check [this](https://api.flutter.dev/flutter/foundation/describeEnum.html) and let's discuss after if it is needed